imagepipeline/src/main/java/com/facebook/imagepipeline/core/ImagePipelineConfig.java

```diff
@@ -699,6 +707,11 @@ public Builder setEncodedMemoryCache(
       return this;
     }
 
+    public Builder setSerialExecutorService(@Nullable SerialExecutorService serialExecutorService){
```

Comment:
The name here is very generic, so it's hard to see what this is used for. Can you name this something like setExecutorServiceForAnimatedImages or similar (maybe even include `preparing`).